packages/web/src/scss/components/Grid/_Grid.scss

```
@@ -24,5 +24,25 @@
     }
 }
 
+@each $breakpoint-name, $breakpoint-value in theme.$breakpoints {
+    $infix: breakpoint.get-modifier('infix', $breakpoint-name, $breakpoint-value);
+
+    @include breakpoint.up($breakpoint-value) {
+        @each $alignment-name, $alignment-value in theme.$alignments-x {
+            .Grid--#{$infix}alignmentX-#{$alignment-name} {
```

While I realize we have `Grid--cols-<X>`, I think this syntax is incompatible with our Stylelint rules (which we cannot see here because the class names are generated). This should be correct:

```scss
@use './string' as spirit-string;
```

```suggestion
            .Grid--#{$infix}#{spirit-string.convert-kebab-case-to-camel-case(alignment-x-#{$alignment-name})} {
```

Output for `left`:

```css
.Grid--alignmentXLeft
```

Anyway, I'm creating a generator for responsive alignment during the work on the new `Flex` component so I expect a discussion to come up.

---

`Flex` PR: https://github.com/lmc-eu/spirit-design-system/pull/1576